java-checks/src/main/java/org/sonar/java/checks/EnumMutableFieldCheck.java

```diff
@@ -65,6 +64,25 @@ public void visitNode(Tree tree) {
       }
     }
   }
+  
+  private static boolean isNotStaticOrFinal(VariableTree variableTree, ModifiersTree modifiersTree) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused method parameter "variableTree". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1172)